packages/kbn-es-query/src/kuery/index.ts

```diff
@@ -6,7 +6,21 @@
  * Side Public License, v 1.
  */
 
+import { estypes } from '@elastic/elasticsearch';
+import { toElasticsearchQuery as astToElasticsearchQuery } from './ast';
+
+/**
+ * @params {String} indexPattern
+ * @params {Object} config - contains the dateFormatTZ
+ *
+ * IndexPattern isn't required, but if you pass one in, we can be more intelligent
+ * about how we craft the queries (e.g. scripted fields)
+ */
+export const toElasticsearchQuery = (...params: Parameters<typeof astToElasticsearchQuery>) => {
```

Comment:
Done to expose correct types!
ast.toElasticsearchQuery has messed up types atm, so at least we get the right type externally.